x-pack/plugins/aiops/public/components/change_point_detection/chart_component.tsx

```diff
@@ -0,0 +1,220 @@
+          store: 'appState',
+        },
+      },
+    ],
+    [dataView.id, requestParams.splitField, annotation.group_field]
+  );
+
+  // @ts-ignore
+  const attributes = useMemo<TypedLensByValueInput['attributes']>(() => {
+    return {
+      title: annotation.group_field,
+      description: '',
+      visualizationType: 'lnsXY',
+      type: 'lens',
+      references: [
+        {
+          type: 'index-pattern',
+          id: dataView.id!,
+          name: 'indexpattern-datasource-layer-2d61a885-abb0-4d4e-a5f9-c488caec3c22',
+        },
+        {
+          type: 'index-pattern',
+          id: dataView.id!,
+          name: 'xy-visualization-layer-8d26ab67-b841-4877-9d02-55bf270f9caf',
+        },
+      ],
+      state: {
+        visualization: {
+          legend: {
+            isVisible: false,
+            position: 'right',
+          },
+          valueLabels: 'hide',
+          fittingFunction: 'None',
+          axisTitlesVisibilitySettings: {
+            x: true,
+            yLeft: true,
+            yRight: true,
+          },
+          tickLabelsVisibilitySettings: {
+            x: true,
+            yLeft: true,
+            yRight: true,
+          },
+          labelsOrientation: {
+            x: 0,
+            yLeft: 0,
+            yRight: 0,
+          },
+          gridlinesVisibilitySettings: {
+            x: true,
+            yLeft: true,
+            yRight: true,
+          },
+          preferredSeriesType: 'line',
+          layers: [
+            {
+              layerId: '2d61a885-abb0-4d4e-a5f9-c488caec3c22',
+              accessors: ['e9f26d17-fb36-4982-8539-03f1849cbed0'],
+              position: 'top',
+              seriesType: 'line',
+              showGridlines: false,
+              layerType: 'data',
+              xAccessor: '877e6638-bfaa-43ec-afb9-2241dc8e1c86',
+            },
+            ...(annotation.timestamp
+              ? [
+                  {
+                    layerId: '8d26ab67-b841-4877-9d02-55bf270f9caf',
+                    layerType: 'annotations',
+                    annotations: [
+                      {
+                        type: 'manual',
+                        label: annotation.label,
+                        icon: 'triangle',
+                        textVisibility: true,
+                        key: {
+                          type: 'point_in_time',
+                          timestamp: annotation.timestamp,
+                        },
+                        id: 'a8fb297c-8d96-4011-93c0-45af110d5302',
+                        isHidden: false,
+                        color: '#F04E98',
+                        lineStyle: 'solid',
+                        lineWidth: 2,
+                        outside: false,
+                      },
+                    ],
+                    ignoreGlobalFilters: true,
+                  },
+                ]
+              : []),
+          ],
+        },
+        query: {
+          query: '',
```

Comment:
Do we need to consider custom queries from the query bar here?

Reply:
not sure it's needed, because the charts are rendered based on the aggregation results, and the change point agg query takes it into account